package/gluon-core/luasrc/usr/lib/lua/gluon/util.lua

```diff
@@ -73,6 +74,20 @@ function node_id()
 	return string.gsub(sysconfig.primary_mac, ':', '')
 end
 
+function find_mesh_devices()
+	local uconn = ubus.connect()
+	if not uconn then
+	    error("Failed to connect to ubusd")
+	end
```

Comment:
I'd prefer the connection to be provided as an argument instead of creating a new one for each call.